libsam.c

```diff
@@ -1559,7 +1734,7 @@ void sam_list_groups(struct hive *hdesc, int listmembers, int human) {
 	cheap_uni2ascii((char *)cd + grpnamoffs, groupname, grpnamlen);
 	
 	if (human) printf("=== Group #%4x : %s\n",grp,groupname);
-	else if (!listmembers) printf("%x:%s:%d\n",grp,groupname,cd->grp_members);
+    else if (!listmembers) printf("%x:%s:%d\n",grp,groupname,cd->grp_members);
```

Comment:
Identation fixes should be in a third commit different from the useful commit and the typos one.